src/database/remove-remote.ts

```diff
@@ -0,0 +1,126 @@
+"use strict";
+
+import { Response } from "request";
+import * as request from "request";
+import responseToError = require("../responseToError");
+import utils = require("../utils");
+import FirebaseError = require("../error");
+import logger = require("../logger");
+import api = require("../api");
+
+class RemoveRemote {
+  private instance: string;
+
+  constructor(instance: string) {
+    this.instance = instance;
+  }
+
+  public deletePath(path: string): Promise<boolean> {
+    return new Promise((resolve, reject) => {
+      const url =
+        utils.addSubdomain(api.realtimeOrigin, this.instance) + path + ".json?print=silent";
+      const reqOptions = {
+        url,
+        json: true,
+      };
+      return api.addRequestHeaders(reqOptions).then((reqOptionsWithToken) => {
+        request.del(reqOptionsWithToken, (err: any, res: Response, body: any) => {
```

Comment:
`err` has type `Error`

```suggestion
        request.del(reqOptionsWithToken, (err: Error, res: Response, body: any) => {
```